subscriptions/spec.md

```diff
@@ -756,10 +756,12 @@ When encoded in JSON, a filter is encoded as follows:
 { "dialect URI-Reference" : { <dialect-specific-properties> } }
 ```
 
-This specification defines the following 7 filter dialects that all
-implementations MUST support:
+###### 3.2.4.1.1 REQUIRED Filter Dialects
 
-###### `exact` filter dialect
+This specification defines the following 6 filter dialects that MUST be 
```

Comment:
Minor thing but can you remove the "6"? I'm pretty sure at some point we'll add or remove one and forget to change the number 🙂